### PR TITLE
chore: rename SimpleCacheClient to CacheClient

### PR DESCRIPTION
**Pull request by nand4011**

### CODE REVIEW

File: momento-sdk/src/intTest/java/momento/sdk/BaseTestClass.java [hunk 1]
```diff
@@ -19,8 +19,7 @@ static void beforeAll() {
   }
 
   private static void ensureTestCacheExists() {
-    try (SimpleCacheClient client =
-        SimpleCacheClient.builder(System.getenv("TEST_AUTH_TOKEN"), 10).build()) {
+    try (CacheClient client = CacheClient.builder(System.getenv("TEST_AUTH_TOKEN"), 10).build()) {
```
Comment (rishtigupta) [inline momento-sdk/src/intTest/java/momento/sdk/BaseTestClass.java:22]: Just FYI - You might need a rebase if I push the Duration changes before this PR is merged!

**Review by rishtigupta — COMMENTED** [review]
Or let me know if you merge this before ["update the ttl type to duration" PR](https://github.com/momentohq/client-sdk-java/pull/176). I can rebase it and push another rev of the PR.

**Review by rishtigupta — APPROVED** [review]

File: momento-sdk/src/intTest/java/momento/sdk/BaseTestClass.java (hunk 1 above)
Comment (cprice404) [reply momento-sdk/src/intTest/java/momento/sdk/BaseTestClass.java:22]: this change is simpler than the duration one, so I think we should merge the duration one first and then rebase this one afterward.

**Review by cprice404 — APPROVED** [review]
lgtm but my rec would be to let the duration change go in before this one.

**Review by rishtigupta — APPROVED** [review]

**Pull request merged**